ppdet/modeling/head/rpn_head.py

```diff
@@ -87,28 +89,26 @@ def forward(self, inputs, feats):
 
     def get_loss(self, loss_inputs):
         # cls loss
-        score_tgt = fluid.layers.cast(
+        score_tgt = paddle.cast(
             x=loss_inputs['rpn_score_target'], dtype='float32')
         score_tgt.stop_gradient = True
-        loss_rpn_cls = fluid.layers.sigmoid_cross_entropy_with_logits(
-            x=loss_inputs['rpn_score_pred'], label=score_tgt)
-        loss_rpn_cls = fluid.layers.reduce_mean(
-            loss_rpn_cls, name='loss_rpn_cls')
+        loss_rpn_cls = ops.sigmoid_cross_entropy_with_logits(
+            input=loss_inputs['rpn_score_pred'], label=score_tgt)
+        loss_rpn_cls = paddle.mean(loss_rpn_cls, name='loss_rpn_cls')
 
         # reg loss
-        loc_tgt = fluid.layers.cast(
-            x=loss_inputs['rpn_rois_target'], dtype='float32')
+        loc_tgt = paddle.cast(x=loss_inputs['rpn_rois_target'], dtype='float32')
         loc_tgt.stop_gradient = True
-        loss_rpn_reg = fluid.layers.smooth_l1(
-            x=loss_inputs['rpn_rois_pred'],
-            y=loc_tgt,
-            sigma=3.0,
+        loss_rpn_reg = ops.smooth_l1(
+            input=loss_inputs['rpn_rois_pred'],
+            label=loc_tgt,
             inside_weight=loss_inputs['rpn_rois_weight'],
-            outside_weight=loss_inputs['rpn_rois_weight'])
-        loss_rpn_reg = fluid.layers.reduce_sum(loss_rpn_reg)
-        score_shape = fluid.layers.shape(score_tgt)
-        score_shape = fluid.layers.cast(x=score_shape, dtype='float32')
-        norm = fluid.layers.reduce_prod(score_shape)
+            outside_weight=loss_inputs['rpn_rois_weight'],
+            sigma=3.0, )
+        loss_rpn_reg = paddle.sum(loss_rpn_reg)
+        score_shape = score_tgt.shape
```

Comment:
could use paddle.shape and remove paddle.to_tensor

Reply:
done，thanks